stdlib/public/Concurrency/CMakeLists.txt

```diff
@@ -29,6 +29,16 @@ if(SWIFT_CONCURRENCY_USES_DISPATCH)
   endif()
 endif()
 
+# Linux requires us to link an atomic library to use atomics.
+# Frustratingly, in many cases this isn't necessary because the
+# sequence is inlined, but we have some code that's just subtle
+# enough to turn into runtime calls.
+if(SWIFT_HOST_VARIANT STREQUAL "Linux")
```

Comment:
Do you mind adding Android here? It is [the only platform currently distributing an up-to-date Swift 5.4 toolchain for a 32-bit platform, armv7](https://github.com/termux/termux-packages/blob/master/packages/swift/build.sh#L12).

Reply:
@rjmccall @buttaface is it possible to make this change in different PR? I would like to merge this change to unblock nightly toolchain.

Reply:
Sure, no problem. I can always add it later if you need this patch right away.

Reply:
https://github.com/apple/swift/pull/38508